lms/djangoapps/teams/search_indexes.py

```diff
@@ -78,7 +82,11 @@ def engine(cls):
         Return course team search engine (if feature is enabled).
         """
         if cls.search_is_enabled():
-            return SearchEngine.get_search_engine(index=cls.INDEX_NAME)
+            try:
+                return SearchEngine.get_search_engine(index=cls.INDEX_NAME)
+            except ConnectionError as err:
+                logging.error("Error connecting to elasticsearch: %s", err)
+                raise ElasticSearchConnectionError
```

Comment:
Is it important to raise a new exception type? If you want to keep the traceback, just use "raise" to continue to raise the original ConnectionError.